docs/sphinx/hpx-functions.rst

```diff
@@ -0,0 +1,102 @@
+.. _HPX Functions:
+
+=============
+HPX Functions
+=============
+
+This document describes the prototypes, the allowable arguments of the listed
+HPX functions.
+
+async
+=====
+
+Async in C++ allows running asynchronous programming. Async programming is useful
+in compute heavy implementations. Basic async function would look like
+
+.. code-block:: text
+   #include <future>
```

Comment:
This should be `#include <hpx/lcos/future.hpp>`

Reply:
757fd3d.